bedrock/settings/base.py

```diff
@@ -1018,6 +1018,11 @@ def get_app_name(hostname):
     },
 }
 
+# Twilio
+TWILIO_ACCOUNT_SID = config("TWILIO_ACCOUNT_SID", default="")
+TWILIO_AUTH_TOKEN = config("TWILIO_AUTH_TOKEN", default="")
+TWILIO_MESSAGING_SERVICE_SID = config("TWILIO_MESSAGING_SERVICE_SID", default="")
```

Comment:
I guess we'll need a corresponding addition to www-config, or are there keys there from previous eras?

Reply:
No keys that I know of, so I'll need to update that. Thanks for the call out.